src/emitter.c

```diff
@@ -1825,6 +1827,7 @@ yaml_emitter_write_indicator(yaml_emitter_t *emitter,
 
     emitter->whitespace = is_whitespace;
     emitter->indention = (emitter->indention && is_indention);
+/*    emitter->open_ended = 0; */
```

Comment:
The test suite is passing although this was commented out. That seems suspicious.

Reply:
That was a change that exists upstream that I'm trying to harmonize. If I leave that in, it breaks tests. I suspect it may be a reverted change from earlier and can be safely removed.

Reply:
I removed this.